src/SFA.DAS.EAS.Portal.UnitTests/Portal/Application/Commands/Paye/WhenPayeSchemeAdded.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EAS.Portal.Application.AccountHelper;
+using SFA.DAS.EAS.Portal.Application.Commands.Paye;
+using SFA.DAS.EAS.Portal.Application.Services;
+using SFA.DAS.EAS.Portal.Client.Database.Models;
+using SFA.DAS.EAS.Portal.Client.Types;
+
+namespace SFA.DAS.EAS.Portal.UnitTests.Portal.Application.Commands.Paye
+{
+    [Parallelizable]
+    [TestFixture]
+    class WhenPayeSchemeAdded
+    {
+        PayeSchemeAddedCommandHandler _sut;
+        Mock<IAccountHelperService> _accountHelperServiceMock;
+        Mock<IAccountDocumentService> _accountServiceMock;
+        AccountDocument _accountDoc;
+        IFixture Fixture = new Fixture();
+
+        [SetUp]
+        public void SetUp()
+        {
+            _accountServiceMock = new Mock<IAccountDocumentService>();
+            _accountHelperServiceMock = new Mock<IAccountHelperService>();
+            _sut = new PayeSchemeAddedCommandHandler(_accountHelperServiceMock.Object, _accountServiceMock.Object);
+
+            Fixture.Customize<Account>(a => a
```

Comment:
Another way this could be written, which could be argued is slightly simpler...
```
    _accountDoc = Fixture.Create<Account>();
    _accountDoc.Id = 1;
    _accountDoc.PayeSchemes = null;
```

Reply:
Removed the PayeSchemes bit for the slight performance boost